quran/quran.com-frontend-next
# Add ability to bookmark verses

### Summary
This PR add the ability to bookmark a verse. The latest 10 bookmarks will appear on the homepage.

### Test Plan
1. Switch to TranslationView.
2. Click on the Verse's action menu and click on Bookmark action.
3. Go to the homepage and it should show there.
4. Refresh the page and the same bookmarked verses should show.
4. Go back to the same verse and click Bookmarked which should remove the bookmarking of the verse.
5. Go to the homepage again and the verse shouldn't show in the bookmarked verses section.

### Screenshots

| Before | After |
| ------ | ------ |
| <img width="1440" alt="Screen Shot 2021-08-19 at 08 42 00" src="https://user-images.githubusercontent.com/15169499/129996000-06f90378-8c2b-4869-9647-133a8bdf5b1c.png"> | <img width="1440" alt="Screen Shot 2021-08-19 at 08 41 46" src="https://user-images.githubusercontent.com/15169499/129995989-95c41a43-bffe-4423-b432-33c5e4f0ada3.png"> |
|<img width="182" alt="Screen Shot 2021-08-19 at 08 44 22" src="https://user-images.githubusercontent.com/15169499/129996004-1ff3148f-c93d-4a99-9698-8a1832d11e87.png">|<img width="182" alt="Screen Shot 2021-08-19 at 08 44 51" src="https://user-images.githubusercontent.com/15169499/129996007-b2c65b9c-7836-4927-9d3a-0ae9e0fd90d0.png">|
||<img width="183" alt="Screen Shot 2021-08-19 at 08 44 37" src="https://user-images.githubusercontent.com/15169499/129996006-f83dc7e7-f2fc-462b-a0b8-6c9e7e9451fc.png">|
